packages/typespec-azure-portal-core/src/decorators.ts

```diff
@@ -15,9 +15,9 @@ import {
   resolvePath,
   validateDecoratorUniqueOnNode,
 } from "@typespec/compiler";
+import * as versioning from "@typespec/versioning";
```

Comment:
try to not do `import *` but import only what you need

Reply:
fixed thank you!!